src/index.js

```diff
@@ -17,41 +17,42 @@ import Player from './Player';
 import Round from './Round';
 import Turn from './Turn';
 import domUpdates from './domUpdates.js';
+import FinalRound from './FinalRound';
+
 
-var boards;
 fetch('https://fe-apps.herokuapp.com/api/v1/gametime/1903/jeopardy/data')
   .then(response => response.json())
   .then(parsedData => getData(parsedData))
   .catch(err => console.error(err));
 
+let answer;
+let boards;
+let game;
+let wager;
+
+// new Audio("https://www.myinstants.com/media/sounds/jeopardy-intro-1.mp3").play()
+
 function getData(info) {
   let clue = new Clue(info);
   boards = clue.makeBoardObject();
 }
 
-let game;
-let answer;
-let wager;
+function checkAnswer(location) {
```

Comment:
Seems like check answer should be the role of a class method. If it remains it `index.js`, then it cannot be tested...